README.md

```diff
@@ -28,199 +28,43 @@ The current TestGround Team is composed of:
 - @raulk - Lead Architect, Engineer, Developer
 - @daviddias - Engineer, Developer, acting as interim PM for the project
 - @jimpick - Engineer, Developer, Infrastructure Lead
+- @nonsense - Engineer, Developer, TestGround as a Service Lead
```

Comment:
@nonsense to clarify (more on your onboarding doc), this translates to:

- Owning TestGround as a Daemon
- Owning TestGround Github Integration
- Ramping up on TestGround Infra so that we can take that off @jimpick's plate (so that then Jim can focus in other things :))

Reply:
//cc @raulk

Reply:
@daviddias OK, makes sense!